_data/education.yml

```
@@ -34,6 +34,16 @@ websites:
       facebook: babbel.languages
       email_address: support@en.babbel.com
 
+    - name: Boston University
+      url: https://www.bu.edu
+      img: bu.png
+      tfa: Yes
+      sms: yes
+      phone: yes
+      software: yes
+      hardware: yes
+      doc: http://www.bu.edu/tech/support/duo/
```

```suggestion
      doc: https://www.bu.edu/tech/support/duo/
```
Please always use https-links if they are available (like in this case).

---

done